docs/CANDIDATE_TECHNOLOGIES.md

```diff
@@ -0,0 +1,159 @@
+# Candidate Technologies for the Gamepad Navigator
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Factors to be Considered](#factors-to-be-considered)
+3. [Candidate Technologies](#candidate-technologies)
+4. [Analysis](#analysis)
+   - [Custom Browser Extension / Add-on](#custom-browser-extension--add-on)
+   - [Electron.js Desktop App](#electronjs-desktop-app)
+   - [Espruino Pico](#espruino-pico)
+   - [Secondary Extensions / Add-ons](#secondary-extensions--add-ons)
+   - [Bookmarklets](#bookmarklets)
+5. [Conclusion](#conclusion)
+
+## Introduction
+
+This document contains the list of technologies, frameworks, and approaches that can be used to implement **Gamepad Navigator**.
+
+## Factors to be Considered
+
+Each of the available technologies has its advantages and disadvantages, and there's a general tendency to prefer the one with more benefits. But the right approach is to choose the technology according to our requirements and the scope of the project.
+
+So, the factors that should be considered for selecting the candidate technology are:
+
+1. Community's adaptability and familiarity with the technology.
+2. Fulfillment of the project's requirements:
+   - Accept and read inputs from the gamepad.
+   - Relay functional code/script into the browser.
+   - Compatible across multiple platforms and devices.
+3. Stability and long-term support for the technology.
+4. Assist in implementing navigation features.
```

Comment:
"Assist in" might be better worded as "Usefulness in".

Reply:
I have reworded this in my previous commit [here](https://github.com/fluid-lab/gamepad-navigator/pull/4/files#diff-b53d05f39c28591f46c423b3aa9c65b5R31).